### PR TITLE
High Scores!

### PR DESCRIPTION
**Pull request by ChielBruin**
Added high scores to the game (and tested it).

<feature

### CODE REVIEW

File: src/main/java/com/github/fishio/control/HighScoreController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	private TableColumn<Entry<String, Integer>, String> playerColumn;
+	@FXML
+	private TableColumn<Entry<String, Integer>, Number> scoreColumn;
+	
+	@Override
+	public void init(Scene scene) {
+		SimpleDoubleProperty p = Settings.getInstance().getDoubleProperty("SCREEN_HEIGHT");
+		p.addListener((o, old, newVal) -> {
+			scrollPane.setPrefHeight(newVal.intValue() - 240);
+		});
```
Comment (LMiljak) [inline src/main/java/com/github/fishio/control/HighScoreController.java]: Consider using a single line for this lambda expression (brackets are unnecesarry).

<layout

**Comment by jessearens** [comment]
Looks good. Great that you also redesigned the main menu to only represent the working functions of our game!

<feature

**Comment by jessearens** [comment]
Oops, I didn't even notice that you forgot to add the tests :O.
However, they also look good!

<testing

File: src/main/java/com/github/fishio/control/HighScoreController.java (hunk 1 above)
Comment (Taeir) [inline src/main/java/com/github/fishio/control/HighScoreController.java]: You could also use

``` java
scrollPane.prefHeightProperty().bind(Settings.getInstance().getDoubleProperty("SCREEN_HEIGHT").subtract(240);
```

<review

File: src/main/java/com/github/fishio/control/SinglePlayerController.java [hunk 2]
```diff
@@ -26,8 +27,11 @@
 import javafx.scene.Node;
 import javafx.scene.Scene;
 import javafx.scene.canvas.Canvas;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Alert.AlertType;
```
Comment (Taeir) [inline src/main/java/com/github/fishio/control/SinglePlayerController.java]: You should remove these (unused) imports

<layout

**Comment by Taeir** [comment]
Apart from the 2 minor things I commented on, I think the HighScores are very well done, good job @ChielBruin!

<feature

**Comment by LMiljak** [comment]
You implemented the suggested changes, so I'm merging it!
:+1: 

<feature

**Pull request merged**